onnxruntime/python/tools/transformers/fusion_gemmfastgelu.py

```diff
@@ -16,8 +17,35 @@
 class FusionGemmFastGelu(Fusion):
```

Comment:
Can we write a test for this one?

Looks like there are no tests for graph transformations in python/tools/transformers, we can a test directory  python/tools/transformers/tests. For new graph transformations we write, we can make sure we have tests.